includes/hooks.php

```diff
@@ -56,6 +56,7 @@ public static function init() {
 			anspress()->add_action( 'ap_vote_removed', __CLASS__, 'update_user_vote_casted_count', 10, 4 );
 			anspress()->add_action( 'ap_display_question_metas', __CLASS__, 'display_question_metas', 100, 2 );
 			anspress()->add_action( 'widget_comments_args', __CLASS__, 'widget_comments_args' );
+			anspress()->add_action( 'ap_breadcrumbs', __CLASS__, 'ap_breadcrumbs' );
```

Comment:
@mebishalnapit Lets do this without a hook.

Reply:
@rahularyan Done.